<launch>

    <arg name="gui"                 		  default="false" />
    <arg name="rviz"                		  default="true" />
    <arg name="nn_model_path"       		  default="$(find rosbot_controller)/nets/new_model_dynamic_batch.onnx"/>
    <arg name="use_nn_model"        		  default="true" />

    <arg name="output_folder"       		  default="/offline_planner_test"/>

    <arg name="file_path"           		  default=""/>
    <arg name="control_mode"        		  default="from_file"/>
    <arg name="desired_number_of_subs"        default="3"/>

    <include file="$(find rosbot_controller)/launch/run_simulation.launch">
        <arg name="rviz"            		  value="$(arg rviz)" />
        <arg name="gui"             		  value="$(arg gui)" />

        <arg name="use_nn_model"    		  value="$(arg use_nn_model)" />
        <arg name="nn_model_path"   		  value="$(arg nn_model_path)" />
    </include>


    <include file="$(find logger)/launch/logger.launch">
        <arg name="output_folder"   		     value="$(arg output_folder)" />
    </include>

    <include file="$(find rosbot_controller)/launch/control_gen.launch">
        <arg name="launch_logger"   			 value="false" />
        <arg name="control_mode"    			 value="$(arg control_mode)" />
        <arg name="file_path"       			 value="$(arg file_path)" />
        <arg name="desired_number_of_subs"       value="$(arg desired_number_of_subs)" />
    </include>

</launch>
